test/test_tensor.py

```diff
@@ -66,6 +66,19 @@ def test_gradcheck(self):
     # coarse approx. since a "big" eps and the non-linearities of the model
     self.assertFalse(gradcheck(tiny_func, tiny_x, eps = 0.1))
 
+  def test_multi_indexing(self):
```

Comment:
indentation. and why are the tests using randomness at all?

Reply:
> I have no idea. Are the tests good and complete?

> indentation. and why are the tests using randomness at all?

for generating shape, and indexes ...

Reply:
I can add hardcode the values, if u want.